source/changelog.txt

```diff
@@ -12,6 +12,25 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.3.0
+------
+
+*Released August 16, 2024*
+
+New features released in this version:
+
+- :issue:`MONGOSH-1550` - Support for Range GA
```

Comment:
**Question**

Can you add a bit more detail here? I am afraid people will not know what Range GA is reading this off, without a link or additional content. 

```suggestion
- :issue:`MONGOSH-1550` - Support for Range GA
```

Reply:
Updated.